res/css/views/dialogs/_SlashCommandHelpDialog.scss

```diff
@@ -0,0 +1,42 @@
+/*!
```

Comment:
You've got some weird copyright going on, which seems duplicated below.

Reply:
Arg this is WebStorm thinking it is helping but it infact is not :(